conftest.py

```diff
@@ -73,6 +74,7 @@
 
 TEAM_MARKERS = {
     "chaos": ["chaos", "deprecated_api"],
+    "gpfs": ["gpfs", "deprecated_api"],
```

Comment:
there's no sub-dir called `gpfs` so this will have no effect and should be removed

Reply:
Removed